examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,279 @@
+/*
+ * This example demonstrates using the pattern matching engine (CuriePME)
+ * to classify streams of accelerometer data from CurieIMU.
+ *** ENHANCEMENT: Automatic data capture with build in MOTION DETECTION functionality, no need of button or any extra input hardware.
+ *
+ * First, the sketch will prompt you to draw some letters in the air (just
+ * imagine you are writing on an invisible whiteboard, using your board as the
+ * pen), and the IMU data from these motions is used as training data for the
+ * PME. Once training is finished, you can keep drawing letters and the PME
+ * will try to guess which letter you are drawing.
+ *
+ * The built in motion detection interrupt is being used to detect whether u are writing or not.
+ * While drawing in the air, the buzzer sounds, so wait until the buzzer turns off after drawing a letter and repeat the cycle.
```

Comment:
Again, is the buzzer needed? I don't think it should be necessary. And a lot more people will be interested in this sketch if it requires no extra hardware.